.travis.yml

```diff
@@ -1,28 +1,25 @@
 language: php
 
-dist: trusty
-sudo: false
+dist: xenial
 
 php:
-  - 5.4
-  - 5.5
   - 5.6
   - 7.0
   - 7.1
   - 7.2
   - 7.3
-  - nightly
-  - hhvm
-  - hhvm-nightly
+  - 7.4
 
 matrix:
   include:
+    - php: hhvm-3.18
+      dist: trusty
```

Comment:
You can safely drop this one, because it's likely to fail all the time now.

Reply:
I don't think so. I intentionally pinned it at a specific version of HHVM 3, so the tests should be stable. It looks like there was only one failure. I'll see if I can pin it at a different version where the test passes. Otherwise, maybe we should simply skip that test in HHVM?

Reply:
Agreed.

Reply:
Doesn't work on HHVM 3.24 or 3.30. I've gone back to 3.18, and marked the test as skipped for HHVM.